src/components/inputs/input-number.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { InputField } from '@dhis2/ui'
+
+export const InputNumber = ({ data, states, onChange }) => (
```

Comment:
This should take `keyDownload`, `maxValue`, `value`, and `disableAll` as props directly, I think.  It also need a PropTypes declaration.